### PR TITLE
CB-10636 Add JSHint for plugins

### PR DESCRIPTION
**Pull request by daserge**
[Jira issue](https://issues.apache.org/jira/browse/CB-10636)

### CODE REVIEW

File: .jscsrc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "disallowMixedSpacesAndTabs": true,
     "disallowTrailingWhitespace": true,
-    "validateLineBreaks": "CRLF",
+    "validateLineBreaks": "LF",
```
Comment (vladimir-kotikov) [inline .jscsrc]: Should we do this validation? AFAIK we had some promlems with line endings only in executable scripts, so CRLF vs LF should n't be a problem for plugin code.

File: package.json [hunk 2]
```diff
@@ -20,8 +20,8 @@
   },
   "scripts": {
     "test": "npm run lint && npm run style",
-    "style": "node_modules/.bin/jscs tests/tests.js",
-    "lint": "node_modules/.bin/jshint tests/tests.js"
+    "lint": "node node_modules/jshint/bin/jshint www && node node_modules/jshint/bin/jshint src && node node_modules/jshint/bin/jshint tests",
```
Comment (vladimir-kotikov) [inline package.json]: Just a nitpick - you can omit path to jshint executable, since npm magic [does this for you](https://docs.npmjs.com/cli/run-script)

**Comment by vladimir-kotikov** [comment]
LGTm

**Comment by daserge** [comment]
Addressed, thanks @vladimir-kotikov

**Pull request merged**